static/src/javascripts/projects/membership/contributions-recurring-tab.js

```diff
@@ -0,0 +1,192 @@
+// @flow
+import bean from 'bean';
+import $ from 'lib/$';
+import fetch from 'lib/fetch';
+import config from 'lib/config';
+import reportError from 'lib/report-error';
+import formatters from 'membership/formatters';
+import { display } from 'membership/stripe';
+
+const CARD_DETAILS = '.js-contribution-card-details';
+const PAYPAL = '.js-contribution-paypal';
+const MANAGE_CARD_LAST4 = '.js-manage-account-card-last4';
+const PAYPAL_EMAIL_ADDRESS = '.js-paypal-email';
+const PAYPAL_SHOW_EMAIL_BUTTON = '.js-show-paypal-button';
+const PAYPAL_HIDE_EMAIL_BUTTON = '.js-hide-paypal-button';
+const PAYPAL_SHOW_EMAIL_MESSAGE = '.js-paypal-email-message';
+// const PACKAGE_COST = '.js-mem-package-cost';
```

Comment:
This is probably on your to-do list already, but we should make sure to remove all of these membership related references/comments before merging this.

Reply:
I _think_ they're all gone now. Will re-check prior to merging anyway.